### PR TITLE
fix make

### PR DESCRIPTION
**Pull request by merlin0x**
```
depklite.h:13:33: error: unknown type name ‘GetNextBit_Data’
   13 | int printBitData(int byteIndex, GetNextBit_Data data);
      |                                 ^~~~~~~~~~~~~~~
make: *** [<builtin>: depklite] Error 1
```

### CODE REVIEW

File: depklite.h [hunk 1]
```diff
@@ -5,6 +5,19 @@
 #include <stdio.h>
 #include <stdint.h>
 
+typedef struct
```
Comment (hackerb9) [inline depklite.h:9]: Why did you move the struct definitions but not their comments?

**Comment by hackerb9** [comment]
Thanks for the fix!

File: depklite.h (hunk 1 above)
Comment (merlin0x) [reply depklite.h:9]: The comments described the functions, not structures :) You are right, comments should be write.
